### DISCUSSION TITLE
Kubernetes와 Docker로 한 번에 끝내는 컨테이너 기반 MSA 질문입니다.

### DISCUSSION THREAD

**Discussion by wand96** [tedilabs/fastcampus-devops#72] [Q&A] [answered]
Part1. Docker 기초 - 02. docker 및 docker-compose설치(Ubuntu) 에서 install-docker.sh와 isntall-docker-compose.sh 두개의 파일을 만들어서 실행하게 되는데 install-docker-compose.sh는 ./install-docker-compose.sh 명령어를 이용하여 실행하였을때 문제없이 실행되는데 install-docker-compose.sh를 실행하려고 하면 bad interpreter: text file busy 라는 오류가 자꾸 뜨더라고요. 이런 오류가 뜨는 이유에 대해 궁금해서 질문 남깁니다. 우선 install-docker같은 경우에는 그냥 명령어 입력해서 설치는 했습니다.

**Comment by posquit0** [comment, marked as answer]
안녕하세요..! 제가 질문의 내용을 정확하게 이해를 하지 못했습니다. 혹시 좀 더 구체적으로 설명해주신다면, 다시 한 번 확인해보도록 하겠습니다.
제가 Step by Step으로 따라 했을 때 동일한 오류 메시지를 확인할 수 있는 형태로 가이드를 주시면 좋습니다.
